ste/xfer.go

```diff
@@ -76,15 +76,7 @@ func computeJobXfer(fromTo common.FromTo, blobType common.BlobType) newJobXfer {
 	case common.EFromTo.BlobLocal(): // download from Azure Blob to local file system
 		return parameterizeDownload(remoteToLocal, newBlobDownloader)
 	case common.EFromTo.LocalBlob(): // upload from local file system to Azure blob
-		switch blobType {
-		case common.EBlobType.None(),
-			common.EBlobType.BlockBlob():
-			return parameterizeSend(anyToRemote, newBlockBlobUploader, newLocalSourceInfoProvider)
-		case common.EBlobType.PageBlob():
-			return parameterizeSend(anyToRemote, newPageBlobUploader, newLocalSourceInfoProvider)
-		case common.EBlobType.AppendBlob():
-			return parameterizeSend(anyToRemote, newAppendBlobUploader, newLocalSourceInfoProvider)
-		}
+		return parameterizeSend(anyToRemote, newBlobUploader, newLocalSourceInfoProvider)
```

Comment:
I haven't tested it to check, but it _looks_ like this change has broken the ability to specify a specific blob type on the command line, for uploads from local disk. (Because it ignores the blobType param that is passed in to computeJobXfer).  Can you test to check whether it has indeed broken that ability?

If it has, then we should consider a few things:
1. Obviously we need it to work
2. But should it be made to work the way it did here... OR
3. Should we align it more closely to the way the same command-line parameter data is used for S2S copying (which respects the command line param, but by a different mechanism in the form of `jptm.BlobTypeOverride()` . That routine ultimately reads the same command line-supplied value data , namely `plan.DstBlobData.BlobType`)

I'll put a suggestion about this in my overall comment on the PR.

Reply:
I did actually test this by hand back when I was building this. Explicit blob types are not broken, and inference does not override them.